BootloaderCommonPkg/Library/IppCryptoLib/rsa_verify.c

```diff
@@ -157,6 +286,23 @@ RsaVerify_Pkcs_1_5 (CONST PUB_KEY_HDR *PubKeyHdr, CONST SIGNATURE_HDR *Signature
                   ((SignatureHdr->SigSize != RSA2048_MOD_SIZE) && (SignatureHdr->SigSize != RSA3072_MOD_SIZE))) {
     return RETURN_INVALID_PARAMETER;
   } else {
-    return VerifyRsaSignature (PubKeyHdr, SignatureHdr, Hash) ? RETURN_SECURITY_VIOLATION : RETURN_SUCCESS ;
+    return VerifyRsaPkcs1Signature (PubKeyHdr, SignatureHdr, Hash) ? RETURN_SECURITY_VIOLATION : RETURN_SUCCESS ;

```

Comment:
VerifyRsaPkcs1Signature returns int type. Better to check the return value with ippStsNoErr.

Reply:
ippStsNoErr defitions are limited to IPP Crypto lib. 
ippStsNoErr would be same as RETURN_SUCCESS

Reply:
Understood. It returns int type, but the if condition looks like BOOLEAN. Assigning it to EFI_STATUS would be better.

Reply:
Aiden, I would address in another patch,